site/docs/usage/command-line.md

```diff
@@ -25,38 +25,40 @@ The `promptfoo` command line utility supports the following subcommands:
-| `--var <key=value>`                 | Set a variable in key=value format                                                                                                                                                                 |
-| `--verbose`                         | Show debug logs                                                                                                                                                                                    |
-| `--watch`                           | Watch the config and prompt files for changes                                                                                                                                                      |
-| `-a, --assertions <path>`           | Path to [standalone assertions](/docs/configuration/expected-outputs/#running-assertions-directly-on-outputs) file                                                                                 |
-| `-c, --config <path>`               | Path to one or more [configuration files](/docs/configuration/guide). `promptfooconfig.js/json/yaml` is automatically loaded if present. Wildcards and directories are supported.                  |
-| `-j, --max-concurrency <number>`    | Maximum number of concurrent API calls                                                                                                                                                             |
-| `-n, --filter-first-n`              | Run the first N test cases                                                                                                                                                                         |
-| `-o, --output <paths...>`           | Path to [output file](/docs/configuration/parameters#output-file) (csv, json, yaml, html)                                                                                                          |
-| `-p, --prompts <paths...>`          | Paths to [prompt files](/docs/configuration/parameters#prompts), directory, or glob                                                                                                                |
-| `-r, --providers <name or path...>` | [`openai:chat`][1], [`openai:completion`][1], [`localai:chat:<model-name>`][2], [`localai:completion:<model-name>`][2], or one of the many other permutations per [API providers](/docs/providers) |
-| `-t, --tests <path>`                | Path to [external test file](/docs/configuration/parameters#tests-file)                                                                                                                            |
-
-[1]: /docs/providers/openai
-[2]: /docs/providers/localai
+| Option                              | Description                                                                     |
```

Comment:
Should we split this into multiple tables according to the way you've categorized it in code?  I like what you did in eval.ts